### PR TITLE
Add gfx12 i8/f8/bf8 Logic Yamls and Testcases for Skinny Size

### PR DESCRIPTION
**Pull request by wenchuanchen**

### CODE REVIEW

**Comment by wenchuanchen** [comment]
tox on gfx1201
============ 19 passed, 96 skipped, 4 warnings in 465.00s (0:07:44) ============

**Comment by wenchuanchen** [comment]
gtest on gfx1201

[==========] 26109 tests from 13 test suites ran. (471871 ms total)
[  PASSED  ] 26109 tests.
hipBLASLt version: 1100

**Comment by wenchuanchen** [comment]
[tox.log](https://github.com/user-attachments/files/17628339/tox.log)
[test.log](https://github.com/user-attachments/files/17628342/test.log)

File: clients/gtest/matmul_gtest.yaml [hunk 1]
```diff
@@ -57,6 +57,24 @@ Tests:
...
+- name: matmul_one_integer_precisions_i8
+  category: quick
+  function:
+    matmul: *integer_precisions_i8
+  matrix_size: *one_matrix_size_range
+  transA_transB: *transA_transB_range
+  alpha: [ 0.0, 1.0 ]
+  beta: [ 0.0, 2.0 ]
+
+- name: matmul_one_real_precisions_1b
```
Comment (jichangjichang) [inline clients/gtest/matmul_gtest.yaml]: only few  arch support FP8/BF8
Need to specify the testing arch

**Comment by wenchuanchen** [comment]
gtest and tox are all pass on gfx1201 with commit 3e696055ebb66f77c6389afceae6462ed413b7df
[test.log](https://github.com/user-attachments/files/17671971/test.log)
[tox.log](https://github.com/user-attachments/files/17671972/tox.log)

**Comment by wenchuanchen** [comment]
add arch=gfx120[0-1] to new i8/f8/b8 testcases

**Review by cmingch — COMMENTED** [review]
why need to separate one_matrix_size_range and skinny_matrix_size_range?

**Comment by wenchuanchen** [comment]
merge one_matrix_size and skinny_matrix_size together
testcase matmul_one is all pass on gfx942

[----------] Global test environment tear-down
[==========] 1488 tests from 1 test suite ran. (33843 ms total)
[  PASSED  ] 1488 tests.
hipBLASLt version: 1100

**Review by cmingch — APPROVED** [review]

**Review by jichangjichang — APPROVED** [review]

**Pull request merged**